target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -198,6 +198,10 @@ tplink,archer-mr200)
 	ucidef_set_led_netdev "wan" "wan" "$boardname:white:wan" "usb0"
 	set_wifi_led "$boardname:white:wlan"
 	;;
+tplink,re200-v1)
+	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "eth0"
+	set_wifi_led "$boardname:green:wlan"
```

Comment:
Can you use a DTS trigger instead?

Reply:
For WiFi I should've fixed it, how to do it for LAN?

Reply:
LAN has to be kept here.